### ISSUE TITLE
1.4.0 doesn't support python 3.5.2 and below

### ISSUE THREAD

**Issue by vedran** [cloudpipe/cloudpickle#360]
Since cloudpickle 1.4.0 uses ClassVar from [this commit](https://github.com/cloudpipe/cloudpickle/commit/1ba10a5491d3f7b788d36c0d3cacc77bd41668a0#diff-d2a3618afedd4e124c532151eedbae09R64), which is only supported by [Python 3.5.3 and above](https://docs.python.org/3/library/typing.html#typing.ClassVar).

I'm amateur python developer, so I'm not sure of the best way to address / announce this. But this may be worth fixing because Ubuntu 16.04 ships with python 3.5.1-2, so all users of that version of Ubuntu would be affected as well.

Thanks!

**Comment by pierreglaser** [comment]
I was not aware of that, thanks for reporting this. We'll land a fix ASAP

**Comment by vedran** [comment]
Thanks for the fast response!

**Comment by pierreglaser** [comment]
Actually, there's more than `ClassVar`: running `cloudpickle`'s test suite using `Python 3.5.0` makes 
`test_generic_type` fail with:
```python-traceback
AttributeError: type object 'Generic' has no attribute '__args__'
```

**Comment by pierreglaser** [comment]
@vedran I just released cloudpickle 1.4.1 which should be compatible with all `Python 3.5` versions.

**Comment by vedran** [comment]
Thanks!
